lib/stateManager.js

```diff
@@ -11,28 +11,22 @@ const rlp = utils.rlp
 
 module.exports = StateManager
 
-function StateManager (opts) {
+function StateManager (opts = {}) {
   var self = this
 
-  var trie = opts.trie
-  if (!trie) {
-    trie = new Trie(trie)
-  }
-
-  var blockchain = opts.blockchain
-  if (!blockchain) {
-    blockchain = fakeBlockchain
-  }
-
-  self.blockchain = blockchain
-  self.trie = trie
+  self.blockchain = opts.blockchain || fakeBlockchain
+  self.trie = opts.trie || new Trie()
   self._storageTries = {} // the storage trie cache
   self.cache = new Cache(trie)
```

Comment:
Should ``cache`` also be made an option, so that people can easier switch that if desired?

Reply:
Yes, but I'd do it in a separate work. `_storageTries` is another cache too, so not sure how it works yet.